telegram/ext/utils/promise.py

```diff
@@ -106,6 +109,18 @@ def result(self, timeout: float = None) -> Optional[RT]:
             raise self._exception  # pylint: disable=raising-bad-type
         return self._result
 
+    def add_done_callback(self, callback: Callable) -> None:
+        """
+        Callback to be run when :class:`telegram.ext.utils.promise.Promise` becomes done.
```

Comment:
Please document that the callback will be given `self.result()` as only positional arg